spec/models/publication_spec.rb

```diff
@@ -323,7 +323,7 @@
 
```

Comment:
I don't really get the whole point of this commit.  What's the point of remove a param value?  It seems like adding value to the app would be to add a test for it, rather than remove it completely.  (I may be missing some context here or just don't get the point of these changes.)

Reply:
I suppose I'd prefer to see `some where` become a `let(:provenance) { 'some provenance' }` or have it actually explicitly use and create unit tests around the default value or changes to the default value.

Reply:
It's a setup for a later commit that removes the 3rd param because it isn't actually used.  These tests aren't testing with that param, so it shouldn't be contentious to tailor them to what they *are* testing.